conda-forge.yml

```diff
@@ -7,8 +7,8 @@ bot:
   - 7.0.x
 provider:
   linux_aarch64: default
-  linux_ppc64le: default
 build_platform:
+  linux_ppc64le: linux_64
```

Comment:
Appears even just moving `linux_ppc64le` back to cross-compiling breaks re-rendering 😞